test/gui/shared/steps/steps.py

```diff
@@ -399,14 +400,63 @@ def createFolder(context, foldername, username=None):
     path = join(syncPath, foldername)
     os.makedirs(path)
 
-
+ 
+@When('user "|any|" creates "|any|" files inside the folder "|any|"')
+def step(context, username, filenumber, foldername):
+    syncPath = context.userData['currentUserSyncPath']
+    path = join(syncPath, foldername)
+    snooze(5)
+    for i in range(0, builtins.int(filenumber)):
+        file_name = f"file{i}.txt"
+        file = open(f'{path}/file{i}', "wb")
+        file.seek(1048576)
```

Comment:
We might want flexibility on the size of file. So instead of hardcoding the number, can you put it in some variable. It would be nice if you can get this data from the feature file.